teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unkown', parent_permission = 'true')
```

Comment:
- Let's update the constructor to ` def initialize(specialization, age, name = 'Unknown')` as we only need this attribute for the teacher since the teacher is above 18 and the method ` def can_use_services?` means there's no age limitation to the teacher, therefore we don't need that parent_permission foe the teacher entity.